src/frequenz/client/dispatch/test/_service.py

```diff
@@ -175,7 +175,10 @@ async def UpdateMicrogridDispatch(
                             )[:]
 
         dispatch = Dispatch.from_protobuf(pb_dispatch)
```

Comment:
Another approach here might be to add an option here.

```py
def from_protobuf(cls, pb_object: PBDispatch, *, set_update_time: bool = True) -> Dispatch:
    ...
```

But then I don't understand why you change the update time that comes from the API.

Reply:
> why you change the update time that comes from the API.

Because we're in the `test` module ;)

Reply:
Ah, I missed that detail, then `replace()` is perfectly fine, sorry for the noise!